.travis.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       - ganache-cli -d --defaultBalanceEther 10000 -h 0.0.0.0 --noVMErrorsOnRPCResponse > /dev/null &
       - sleep 60
     script:
-      - yarn test
+      - yarn cpk:test
```

Comment:
just to double-check: it will only test the CPK package and not other packages in the monorepo, is this what you were trying to achieve?

Reply:
yes